Release v3.4.1 "make check" fails

Got this on Rasbian (Debian 7):
`g++ -DHAVE_CONFIG_H -I. -I..  -I./../gmock/gtest/include -I./../gmock/include  -pthread -DHAVE_PTHREAD=1 -DHAVE_ZLIB=1 -Wall -Wno-sign-compare -g -DNDEBUG -MT google/protobuf/testing/protobuf_test-file.o -MD -MP -MF google/protobuf/testing/.deps/protobuf_test-file.Tpo -c -o google/protobuf/testing/protobuf_test-file.o `test -f 'google/protobuf/testing/file.cc' || echo './'`google/protobuf/testing/file.cc`
`google/protobuf/testing/file.cc: In static member function ?static bool google::protobuf::File::CreateDir(const string&, int)?:`
`google/protobuf/testing/file.cc:124:5: error: ?const string? has no member named ?back?`
`Makefile:7166: recipe for target 'google/protobuf/testing/protobuf_test-file.o' failed`

--- a/src/google/protobuf/testing/file.cc
+++ b/src/google/protobuf/testing/file.cc
@@ -121,7 +121,7 @@ void File::WriteStringToFileOrDie(const string& contents, const string& name) {
 
 bool File::CreateDir(const string& name, int mode) {
   if (!name.empty()) {
-    GOOGLE_CHECK_OK(name.back() != '.');
+    GOOGLE_CHECK_OK(name[name.size()-1] != '.');
   }
   return mkdir(name.c_str(), mode) == 0;
 }

--- a/src/google/protobuf/testing/file.cc
+++ b/src/google/protobuf/testing/file.cc
@@ -121,7 +121,7 @@ void File::WriteStringToFileOrDie(const string& contents, const string& name) {
 
 bool File::CreateDir(const string& name, int mode) {
   if (!name.empty()) {
-    GOOGLE_CHECK_OK(name[name.size()-1] != '.');
+    GOOGLE_CHECK_OK(name[name.size() - 1] != '.');
   }
   return mkdir(name.c_str(), mode) == 0;
 }

